Rakefile

```diff
@@ -70,3 +70,41 @@ task :populate, [:storage_root] => [:environment] do |_t, args|
     puts "You need to enter a specific storage root"
   end
 end
+
+desc "Fire off M2C existence check on a single storage root"
+task :m2c_exist_single_root, [:storage_root, :boolean, :profile] => [:environment] do |_t, args|
+  unless args[:profile] == 'profile' || args[:profile].nil?
+    p "Usage: rake m2c_exist_single_root[fixture_sr1,false] || rake m2c_exist_single_root[fixture_sr1, false, profile]"
+    exit
+  end
+  root = args[:storage_root].to_sym
+  storage_dir = "#{Settings.moab.storage_roots[root]}/#{Settings.moab.storage_trunk}"
+  puts "#{Time.now.utc.iso8601} Running moab to catalog existence check for #{storage_dir}"
```

Comment:
maybe capitalize Moab and Catalog for better skimmability?  Possibly also Existence Check:  `Running Moab to Catalog Existence Check for `